detection-rules/headers_tld_in_hops_with_authfailure_or_credtheft.yml

```diff
@@ -0,0 +1,29 @@
+name: "Suspicious TLD in headers with SPF/DMARC failures or high confidence credential theft"
+description: |
+  Flags inbound emails with suspicious TLDs in the headers, SPF/DMARC failures, or high-confidence credential theft indicators
+type: "rule"
+severity: "medium"
+source: |
+  type.inbound
+  and any(headers.domains, .tld in $suspicious_tlds)
```

Comment:
wyt of adding a "and .tld != sender.email.domain.tld" here? otherwise any message from one of these TLDs with misconfigured SPF or misconfigured gateway would likely trigger

Reply:
sweet. heads up i'm not seeing it committed

Reply:
Whaa? It's there?

Reply:
`and any(headers.domains, .tld in $suspicious_tlds and .tld != sender.email.domain.tld)`